site/content/en/blog/releases/1.27.0-rc.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "1.27.0 - Release Candidate"
+linkTitle: "1.27.0-rc"
+date: "2022-10-19"
+---
+This is the 1.27.0-rc release of Agones.
+
+This release brings us support for go v 1.19.1, so if you are installing this release, remember to upgrade your version.
+
+This release has lots of important changes including:
+- Enable image streaming everywhere by default 
+- An example of setting autoscaler behavior in a Fleet. 
+- Log cleanup for pods not yet running
```

Comment:
```suggestion
In this release Agones upgrades to Go 1.19 (from 1.17!), so we can take advantage of all the [improvements therein](https://tip.golang.org/doc/go1.19).

This release has lots of important changes including:
- Enable image streaming everywhere by default on GKE
- Provide more flexibility on utilising the cluster autoscaler with graceful termination.
- Reduced logging verbosity.
```

I don't think our users don't need to update their version of Go (at least I don't think so?) @gongmax WDYT?

How do we feel about this?